AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
-        static let mic: String = LocalizedStringKey("Phone Mic").stringKey
-        static let addNoteButton: String = LocalizedStringKey("Add a note").stringKey
-    }
-
+        static let measurementsMicText: String = NSLocalizedString("Most recent measurement:",
+                                                                   comment: "")
+        static let stopButton: String = NSLocalizedString("Stop recording",
+                                                          comment: "")
+        static let editButton: String = NSLocalizedString("Edit session",
+                                                          comment: "")
+        static let shareButton: String = NSLocalizedString("Share session",
+                                                           comment: "")
+        static let deleteButton: String = NSLocalizedString("Delete session",
+                                                            comment: "")
+        static let stopRecordingButton: String = NSLocalizedString("Finish recording session",
+                                                                   comment: "")
+        static let enterStandaloneModeButton: String = NSLocalizedString("Enter standalone mode",
+                                                                         comment: "")
+        static let finishAlertTitle: String = NSLocalizedString("Finish recording ",
+                                                                comment: "It consists of few parts. Please consider them together. Whole text is following: Finish recording this session ?")
+        static let finishAlertTitle_2: String = NSLocalizedString("this session",
+                                                                  comment: "")
+        static let finishAlertTitle_3: String = NSLocalizedString("?",
+                                                                  comment: "")
+        static let finishAlertTitle_3_SYNC: String = NSLocalizedString("and sync from SD card?",
+                                                                       comment: "")
+        static let finishAlertMessage_1: String = NSLocalizedString("The session will be moved to ",
+                                                                    comment: "It consists of few parts. Please consider them together. Whole text is following: The session will be moved to Mobile Dormant tab and you won't be able to add new measurement to it.")
+        static let finishAlertMessage_2: String = NSLocalizedString("Mobile Dormant",
+                                                                    comment: "")
+        static let finishAlertMessage_3: String = NSLocalizedString(" tab and you won't be able to add new measurement to it.",
+                                                                    comment: "")
+        static let finishAlertMessage_4: String = NSLocalizedString("SD card will be cleared afterwards",
```

Comment:
all multipart strings here should be merged into single ones using techniques described above